.ci/.jenkins_nodejs.yml

```diff
@@ -1,4 +1,6 @@
 NODEJS_VERSION:
+  - "15"
+  - "15.0"
```

Comment:
How about dropping "15.0" and just have "15" here? I'm inclined to not bother explicitly supporting the first minor-level release of odd (i.e. non-LTS) node releases (https://github.com/nodejs/Release#release-phases). IMO, the main value of supporting the odd-numbered releases is to (a) prepare for changes coming in the subsequent even-numbered LTS release and (b) to help users of this module that are doing the same.

@astorm Do you have an opinion?

Reply:
I don't know if I have a say in this :D However, this seems very reasonable. V15 will not receive any support in 6 months from now anyway